zebra/zebra_vty.c

```diff
@@ -1411,7 +1431,18 @@ DEFPY (show_ip_nht,
 			return CMD_WARNING;
```

Comment:
We should free `json` here, right when returning before?

Reply:
ACK. Good catch!